### PR TITLE
Medical imaging example

### PR DESCRIPTION
**Pull request by gheinrich**

### CODE REVIEW

**Comment by gheinrich** [comment]
@IsaacYangSLA can you review this PR please? Thanks!

**Comment by IsaacYangSLA** [comment]
@gheinrich , you are welcome.  Yes, I will review it.

File: examples/medical-imaging/README.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+```sh
+$ pip install $DIGITS_ROOT/plugins/data/sunnybrook/
+```
+
+Restart DIGITS for the change to take effect.
+
+## Dataset creation
+
+You will need to download two files from the [Sunnybrook cardiac data web site](http://www.cardiacatlas.org/studies/sunnybrook-cardiac-data/):
+- [all the DICOM images](http://capwebprd02.its.auckland.ac.nz/files/SCD/SCD_DeidentifiedImages.zip) (2.6GB)
```
Comment (gheinrich) [inline examples/medical-imaging/README.md]: You're right! I pointed to a different location than that I was using. Let me fix this.

Comment (gheinrich) [inline examples/medical-imaging/README.md]: @IsaacYangSLA I have updated the links, thanks.

**Comment by IsaacYangSLA** [comment]
@gheinrich , thanks.  I think it's good to merge this PR.

**Comment by IsaacYangSLA** [comment]
@lukeyeager , I think we don't have to test it since it worked in the lab.  Reading Greg's codes is like reading a masterpiece, so I believe no problem on the coding side.  Should we merge it?

**Pull request merged**